dnf/cli/output.py

```diff
@@ -1232,12 +1265,53 @@ def _add_line(lines, data, a_wid, po, obsoletes=[]):
                                        remainder_column=2)
             (n_wid, a_wid, v_wid, r_wid, s_wid) = columns
 
+            # Do not use 'Package' without context. Using context resolves
+            # RhBug 1302935 as a side effect.
+            msg_package = select_short_long(n_wid,
+            # Translators: This is the short version of 'Package'. You can
+            # use the full (unabbreviated) term 'Package' if you think that
+            # the translation to your language is not too long and will
+            # always fit to limited space.
+                                            C_('short', 'Package'),
```

Comment:
We are loosing plural forms here

Reply:
We are not losing plurals because don't use them now. ```msg_package``` initialized here replaces the current line [output.py:1236](https://github.com/rpm-software-management/dnf/blob/master/dnf/cli/output.py#L1236):

```python
        self.fmtColumns(((P_('Package', 'Packages', 1), -n_wid),
```

which calls ```P_()``` (select the plural translation) with always ```n = 1``` (always the singular number). See also below.